CMakeLists.txt

```diff
@@ -29,8 +42,6 @@ qt_add_executable(sioyek WIN32 MACOSX_BUNDLE
     pdf_viewer/pdf_renderer.cpp pdf_viewer/pdf_renderer.h
     pdf_viewer/pdf_view_opengl_widget.cpp pdf_viewer/pdf_view_opengl_widget.h
     pdf_viewer/rapidfuzz_amalgamated.hpp
-    pdf_viewer/sqlite3.c pdf_viewer/sqlite3.h
```

Comment:
This now means that we can now remove `sqlite3.c`, `sqlite3.h` and `sqlite3ext.h` from our codebase. But I'll leave it up to you if we decide to move completely to cmake builds for everything